girder_worker/docker/nvidia.py

```diff
@@ -38,7 +38,7 @@ def get_nvidia_configuration():
 
 def is_nvidia_image(api, image):
     labels = api.inspect_image(image).get('Config', {}).get('Labels')
-    return labels and labels.get('com.nvidia.volumes.needed') == 'nvidia_driver'
+    return bool(labels and labels.get('com.nvidia.volumes.needed') == 'nvidia_driver')
```

Comment:
Why is this needed? Isn't == always going to produce a bool?

Reply:
labels will be None if ```api.inspect_image(image)``` returns {'Config': {}},  and in a weird turn of events,  ```None and {True|False}```  returns None. womp. 

```python
In [1]: print(None and True)
None

In [2]: print(None and False)
None

In [3]: print(False and None)
False

In [4]: print(True and None)
None

```

Reply:
What about ```labels is not None``` ?

Reply:
> in a weird turn of events, None and {True|False} returns None. womp.

In both python and javascript, `and` and `or` (`&&` and `||`) have the same behavior:

* `and`: returns the left value if it's falsy, otherwise returns the right value
* `or`: returns the left value if it's truthy, otherwise returns the right value